packages/cerebral-forms/src/factories/validateForm.js

```diff
@@ -37,7 +37,7 @@ export default function validateFormFactory (passedFormPathTemplate) {
       context.state.merge(path, {
         isValid: isValid,
         hasValue: hasValue,
-        errorMessage: isValid ? null : field.errorMessages[result.failedRuleIndex],
+        errorMessage: isValid ? null : (field.errorMessages && field.errorMessages.length > 0 ? field.errorMessages[result.failedRuleIndex] : null),
```

Comment:
Not sure if this is still correct now as errorMessage can also be requiredMessage if field `isRequired && !hasValue`.